src/services/auth.js

```
@@ -16,9 +18,12 @@ const emailSubject = require('~/consts/emailSubject')
 const {
   tokenNames: { REFRESH_TOKEN, RESET_TOKEN, CONFIRM_TOKEN }
 } = require('~/consts/auth')
-const jwt = require('jsonwebtoken');
+const jwt = require('jsonwebtoken')
 const bcrypt = require('bcrypt')
 const SALT_ROUNDS = 10
+if (!process.env.GMAIL_CLIENT_ID) {
+  throw new Error('GMAIL_CLIENT_ID environment variable is required');
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)